core/src/main/resources/lib/hudson/ballColorTd.jelly

```diff
@@ -58,7 +58,7 @@ THE SOFTWARE.
 
         <j:choose>
           <j:when test="${iconClassName != null}">
-            <l:icon class="${iconClassName} icon-lg" alt="${it.description}" tooltip="${it.description}" />
+            <l:icon src="${iconClassName}" class="${iconClassName}" alt="${it.description}" tooltip="${it.description}" />
```

Comment:
We've generally used `clazz` in Jelly for this (with one painful exception IIRC).

Reply:
this isn't a new feature it's the icon class specification that's always been there

Reply:
@janfaracik This line (and this line alone) seems to cause [JENKINS-68270](https://issues.jenkins.io/browse/JENKINS-68270). Reverting this line to its old value fixes JENKINS-68270 for me.

Reply:
I can confirm that the old icon class spec of `gitlab-logo icon-lg` works, while the new icon class spec of `gitlab-logo` (as of the above change) cannot be found in `iconsByCSSSelector`.